driverthroughput/main.py

```diff
@@ -315,5 +354,10 @@ def _create_colocated_throughput_session(cls,
                     database_nodes=nodes,
                     database_cpus=db_cpus,
                     iterations=iterations,
-                    tensor_bytes=_bytes)
-        return model
\ No newline at end of file
+                    tensor_bytes=_bytes,
+                    language=language)
+        return model
+
+if __name__ == "__main__":
+    import fire
```

Comment:
this is probably purely stylistic, but I like having all the imports together at the top of the file

Reply:
done